test/smoke_test/smoke_test.py

```diff
@@ -55,6 +55,20 @@ def check_nightly_binaries_date(package: str) -> None:
                     f"Expected {module['name']} to be less then {NIGHTLY_ALLOWED_DELTA} days. But its {date_m_delta}"
                 )
 
+def cuda_runtime_error():
```

Comment:
Nit: use type annotations, also prefix function with `test_` or `check_` to conform naming standard for the rest of the package
```suggestion
def test_cuda_runtime_errors_captured() -> None:
```